components/Header.js

```diff
@@ -0,0 +1,22 @@
+import Link from "next/link";
+import Router from "next/router";
+import handleClick from "./activeLink";
```

Comment:
You need to import active link not the event handler. That is abstracted by the component.